src/txkube/test/test_authentication.py

```diff
@@ -0,0 +1,125 @@
+# Copyright Least Authority Enterprises.
+# See LICENSE for details.
+
+import os
+
+from uuid import uuid4
+
+from fixtures import TempDir
+
+from testtools.matchers import Equals, Contains
+
+from twisted.python.filepath import FilePath
+from twisted.internet.protocol import Factory
+from twisted.web.http_headers import Headers
+from twisted.test.iosim import ConnectionCompleter
+from twisted.test.proto_helpers import AccumulatingProtocol, MemoryReactor
+
+from ..testing import TestCase
+
+from .. import authenticate_with_serviceaccount
+
+# Just an arbitrary certificate pulled off the internet.  Details ought not
```

Comment:
indicating where it was pulled from (and in which manner) may be useful should the time come to cycle it out with another certificate (cert expiry/domain death/etc)